### PR TITLE
CS: clean some whitespaces/indentation

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Issues        | Fix CS
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php [hunk 1]
```diff
@@ -483,7 +483,7 @@ private function parseFileToDOM(string $file): \DOMDocument
                 }
             }
             if ($errors) {
-                throw new InvalidArgumentException(\sprintf('Unable to parse file "%s": ', $file).implode("/n", $errors), $e->getCode(), $e);
```
Comment (xabbuh) [inline src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:486]: Shouldn’t this have been `\n` instead?

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php (hunk 1 above)
Comment (keradus) [reply src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:486]: I honestly have no clue. I would only guess "yes".
I assume there is some test validating result of current behaviour. If you eager to investigate/fix this, please raise it separately.
Unfortunately, I won't be able to check this on my side shortly, and adjust/add missing tests

**Review by alexandre-daubois — APPROVED** [review]

File: src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php (hunk 1 above)
Comment (OskarStark) [reply src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:486]: * https://github.com/symfony/symfony/pull/58487

**Comment by nicolas-grekas** [comment]
Thank you @keradus.

**Pull request merged**